app/importers/trade_event/ustda_data.rb

```diff
@@ -49,9 +49,12 @@ def import
 
     def events
       fh = open(@resource, 'r:UTF-8')
-      fh.readline # Remove 'Last Updated: ...'
-      content = fh.read.encode('UTF-16le', invalid: :replace, replace: '').encode('UTF-8')
-      doc = CSV.parse(content, headers: true, header_converters: :symbol, encoding: 'UTF-8')
+
+      raw = fh.read.encode('UTF-16le', invalid: :replace, replace: '').encode('UTF-8')
+      rows_as_strings = raw.split(/[\r\n]+/)[1..-1]  # Deal with the absence of newlines, throw away invalid "Last Updated..." first row.
+
+      doc = CSV.parse(rows_as_strings.join("\n"), headers: true, header_converters: :symbol, encoding: 'UTF-8')
+
```

Comment:
@carloslima @jamesisaacs any ideas on how to do this more...elegantly? See [#85847236](https://www.pivotaltracker.com/story/show/85847236).